LongWarOfTheChosen/Src/LW_Overhaul/Classes/LWTemplateMods.uc

```diff
@@ -3479,6 +3481,7 @@ function ReconfigFacilities(X2StrategyElementTemplate Template, int Difficulty)
 			FacilityTemplate.SoldierUnlockTemplates.RemoveItem('MeditationPreparationUnlock');
 			FacilityTemplate.SoldierUnlockTemplates.RemoveItem('ParkourUnlock');
 			FacilityTemplate.SoldierUnlockTemplates.RemoveItem('InfiltrationUnlock');
+			FacilityTemplate.SoldierUnlockTemplates.RemoveItem(default.GTSUnlocksToRemove);
```

Comment:
Do you see the problem here? 😄

Reply:
No, but I'll try my best to figure out what it is.